app/jinja_filters.py

```diff
@@ -546,3 +546,17 @@ def map_list_collector_config(
 @blueprint.app_context_processor
 def map_list_collector_config_processor():
     return dict(map_list_collector_config=map_list_collector_config)
+
+
+@blueprint.app_template_filter()
+def format_paragraphs(text):
```

Comment:
```suggestion
def format_paragraphs(text):
        return "\n".join(f'<p>{paragraph}</p>' for paragraph in text.splitlines())
```

Reply:
Should we not be questioning the need for paragraphs in question descriptions? if this much content is needed for a question are there bigger issues?

Reply:
Can @bameyrick point to an example where this is causing formatting issues at the moment?

Reply:
I couldn't find any examples where there were multiple paragraphs of question description, but I checked with @hamishtaplin whether it was possible to do multiple paragraphs in Author and you can. I thought it would be safest to wrap the description text in paragraph tags here than in the template just incase somebody authors a survey with multiple paragraphs.